opal/util/arch.c

```diff
@@ -67,7 +67,7 @@ static inline int32_t opal_arch_ldisintel( void )
             j = j-1;
         }
     }
-    return (pui[j] & (1 << i) ? 1 : 0);
+    return ((pui[j] & (1u << i)) ? 1 : 0);
```

Comment:
i being capped at 31 the 1u is not necessary.

Reply:
This one concerns me: that's some kind of merge error, I didn't intentionally change that.  After I make the changes to use dt_elem_desc_t I'll have to review my commit more carefully to make sure there's nothing else unintentional in there.